securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java

```diff
@@ -147,6 +147,12 @@ public static void clearAllValues(@NonNull Context context) throws SecureStorage
         clearAllSecureValues(context);
     }
 
+    public static boolean containsKey(@NonNull Context context, @NonNull String key){
```

Comment:
As "containsKey" is confusing in the context of this library I would suggest to stay with the known SharedPreferences nomination --> contains

Reply:
Of course, not a problem.